bedrock/base/templates/macros-protocol.html

```diff
@@ -259,7 +258,9 @@
   {% if card_data -%}
     {% set image = '<img class="mzp-c-card-image" src="' ~ card_data.image_url ~ '" srcset="' ~ card_data.highres_image_url ~ ' 1.5x">' %}
```

Comment:
This still needs some additional logic as we're always setting `srcset` on line 259. So if `highres_image_url` does not exist, the markup ends up as:

`<img class="mzp-c-card-image" src="https://www-dev.allizom.org/media/contentcards/img/home-fr/card_6/misinformation.d8a3af58765b.png" srcset="{{ no such element: dict object['highres_image_url'] }} 1.5x">`